modules/install-upgrade/pages/production-cluster-sizing.adoc

```diff
@@ -0,0 +1,216 @@
+
+* _Average message size (uncompressed)_ - this is the most important number to understand. A message is sized by the number of bytes. The data that composes a message includes a *message key*, *message properties*, and the *message payload*. A *message key* is roughly the same number of characters as a GUID. *Message properties* is a key/value collection of metadata, so the number of characters varies. The *message payload* accounts for the bulk of the sizing variability. To start, assume the message is a JSON string with some number of characters. +
+For more on message compression, see the https://pulsar.apache.org/docs/2.11.x/concepts-messaging/#compression[Pulsar documentation].
+
+* _Incoming message throughput (e.g. # of messages per second)_ - this is the second most important number to understand. Throughput is expressed as a number of messages that the cluster can produce in a second. Think about this number in terms of steady traffic and burst traffic. Pulsar can scale brokers to handle bursts, so you don’t need to size for maximum workload. If you were streaming in data every time someone clicked on a website and the site received a constant 2000 views per second, then your minimum throughput must be able to handle a load above that requirement, because that stream won't be the only load on the cluster.
+
+* _Number of active topics_ - If the applications using the cluster will be event-driven, then there will most likely be quite a few topics, possibly one for every event type. If the cluster is mostly for streaming data, then each data type will have multiple topics supporting it.
+
+* _Message retention and TTL period_ - Assume that most of the topics in the cluster will be persistent, meaning that messages (once acknowledged) are stored for later lookup on disk. The default retention times in Pulsar are XXXX and the default time to live is XXXX. With a feel for average message size and retention policies we can begin to understand storage needs. TTL (time-to-live) determines how long an unacknowledged message lasts before it is deleted, even before it is acknowledged.
+
+* _Tiered storage policies_ - Tiered storage offloads bookkeeper data to cheaper, long-term storage, and can impact cluster sizing. For more on tiered storage, see https://pulsar.apache.org/docs/2.11.x/tiered-storage-overview/[Pulsar documentation].
+
+There are other factors that could be a part of the aggregated cluster workload. As you gain familiarity with Pulsar you can further customize this calculation, but for now, we will estimate with the above numbers to size a cluster.
+
+[#assumptions]
+== Assumptions
+
+The methodology in this guide relies on the assumption that the ratio of the broker instance count to the bookkeeper instance count is static.
+For this example, we're using 1-to-5 as the static broker-to-bookkeeper ratio.
```

Comment:
With respect to static broker-to-bk ratio:

In Astra Streaming production clusters, we have (brokers to BKs) - 3-3, 5-3, 5-6.  Our Azure eastus cluster has the highest messaging workloads (small msg size), this is being used by our customer PEX.   Typically, we have more Brokers than BKs, but it does depend on workloads, and the resources given to the BKs (memory, cpu).  BKs performance is primarily disk-bound.